src/utils/traffic-mesh.js

```diff
@@ -10,6 +10,10 @@ const EXEC_NAME = 'traffic-mesh'
 
 const LATEST_VERSION = 'v0.22.1'
 
+const EDGE_HANDLERS_BUNDLER_CLI_PATH = path.resolve(
+  path.join(path.dirname(require.resolve('@netlify/plugin-edge-handlers')), 'cli.js'),
```

Comment:
Note that using `require.resolve()` might make it harder for this repository to switch to ES modules in the future.

I am wondering: the `plugin-build-edge-handlers` binary should be available in `node_modules/.bin`. This can be retrieved with [`npm-run-path`](https://github.com/sindresorhus/npm-run-path) for example, then added to the `$PATH`, so that the binary can be executed using only the binary name (as opposed to full path).

I am not sure this is a practical solution, and it might complicate things more than help. Just raising this as a possible alternative.

Reply:
I'm trying to avoid putting internal binaries on `PATH`, so I'd prefer passing the full path to the binary to mesh-forward.

I do agree with the last part though, `npm-run-path` might be a more future-proof implementation. I'll look into that.

Reply:
> I'm trying to avoid putting internal binaries on PATH, so I'd prefer that we keep passing the full path to the binary to mesh-forward.

Yes, good point. Also, adding `node_modules/.bin` would also bring the other locally installed Node.js binaries to the `PATH`, which might not be a good side effect. The current solution makes sense in that light :+1:

Reply:
Let's roll the `npm-run-path` changes in another PR. I'd like to move forward with the implementation and then I'll optimize. If that's okay for you -- of course.

Reply:
Just to clarify, I actually think using `npm-run-path` might be a bad idea due to:

> Also, adding node_modules/.bin would also bring the other locally installed Node.js binaries to the PATH, which might not be a good side effect. 

So the current solution looks good to me.

Reply:
Ah, I didn't look at its docs yet. I thought it's just a library that was able to resolve a locally installed node binary, such that we could replace `require.resolve(...)` with a solution that worked even for ES modules.

Reply:
It depends. 
When run in ES modules mode, Node.js [does remove `require.resolve()`](https://nodejs.org/api/esm.html#esm_no_require_resolve). 
However, the experimental `import.meta.resolve()` might be used instead.

Reply:
I'm open to anything when that time comes 🙌